examples/window-app/common/window-app.matter

```diff
@@ -2462,6 +2464,7 @@ endpoint 2 {
     ram      attribute clusterRevision default = 4;
 
     handle command Identify;
+    handle command TriggerEffect;
```

Comment:
This is an optional command.

Reply:
This might be related to the xml change which would make it mandatory
`<command source="client" code="0x40" name="TriggerEffect" optional="true">`